packages/main/src/components/FormGroup/FormGroupTitle.tsx

```diff
@@ -0,0 +1,43 @@
+import { CssSizeVariables, ThemingParameters } from '@ui5/webcomponents-react-base';
+import React, { CSSProperties } from 'react';
+import { createUseStyles } from 'react-jss';
+
+interface FormGroupTitlePropTypes {
+  titleText: string;
+
+  style?: CSSProperties;
+}
+
+const useStyles = createUseStyles(
+  {
+    title: {
+      gridColumn: '1 / span 12',
+      display: 'flex',
+      alignItems: 'center',
+      height: CssSizeVariables.sapWcrFormGroupTitleHeight,
+      lineHeight: CssSizeVariables.sapWcrFormGroupTitleHeight,
+      fontFamily: ThemingParameters.sapFontHeaderFamily,
+      color: ThemingParameters.sapGroup_TitleTextColor,
+      fontSize: ThemingParameters.sapFontHeader6Size,
+      fontWeight: 'bold',
+      backgroundColor: ThemingParameters.sapGroup_TitleBackground,
```

Comment:
This should probably be removed again, right?

https://github.com/SAP/ui5-webcomponents-react/pull/4171

```suggestion
```